### PR TITLE
Formats project with new code style

### PR DESCRIPTION
**Pull request by ericdecanini**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [x] Technical
- [ ] Other :

## Content

Autoformats project

## Motivation and context

Aligns project with new editorconfig standards

This is to prepare for incremental code style changes, and should be the only PR in the code style subproject with a massive diff, thus the greatest challenge.

I tested conflicts for this by merging this branch into the [Tchap fork](https://github.com/tchapgouv/tchap-android) and this seemed to introduce only a few more easy-to-resolve conflicts than there would be when merging with develop

See also https://github.com/vector-im/element-android/pull/5644

## Screenshots / GIFs

N/A

## Tests

- Checkout Tchap fork
- Add the original element repo as upstream and fetch all the branches
- Merge this branch with Tchap develop

You will see that there are a lot of conflicts, but most of these conflicts come from changes that are on our repo's develop and not the changes on this branch specifically. If you repeat the steps above but instead merge upstream/develop into origin/develop, you will see that most of those conflicts are still present and this branch adds only few.

## Tested devices

- [x] Physical
- [ ] Emulator
- OS version(s): Android 10

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt [hunk 1]
```diff
@@ -58,15 +58,15 @@ class SwipeToDismissHandler(
...
+            MotionEvent.ACTION_MOVE -> {
                 if (isTracking) {
                     val translationY = event.y - startY
                     swipeView.translationY = translationY
                     onSwipeViewMove(translationY, translationLimit)
                 }
                 return true
             }
-            else                                             -> {
+            else -> {
```
Comment (bmarty) [inline library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt:69]: I do not understand why we have so many changes.
The plan was to change the rules one by one.
So:
- create the .editorconfig with the current rules, eventually format the whole code base to fix existing issues.
- Change the rules one by one, in .editorconfig and then format, so for instance, change the rule for `arrow alignment` and see in the PR only changes about the arrows.

WDYT?

Comment (ericdecanini) [reply library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt:69]: The format here is actually without any rule changes. The problem we're facing is the app doesn't consistently abide by our currently set rules and breaks many basic lint rules in lots of places

Comment (bmarty) [reply library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt]: The current rule is to align the arrows in a `when` block vertically and here it is changing, so I do not understand.

Comment (ericdecanini) [reply library/attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/SwipeToDismissHandler.kt]: (Updated the PR fixing this. This file no longer has any diffs)

**Review by bmarty — APPROVED** [review]
OK, with this changes

**Pull request merged**